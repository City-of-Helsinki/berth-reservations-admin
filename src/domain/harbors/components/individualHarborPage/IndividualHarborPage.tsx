import React from 'react';

import styles from './individualHarborPage.module.scss';

const IndividualHarborPage: React.SFC = ({ children }) => {
  return <div className={styles.individualHarborPage}>{children}</div>;
};

export default IndividualHarborPage;
