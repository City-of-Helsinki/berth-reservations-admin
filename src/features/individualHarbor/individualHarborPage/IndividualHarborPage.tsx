import React from 'react';
import { useTranslation } from 'react-i18next';

import styles from './individualHarborPage.module.scss';
import PageTitle from '../../../common/pageTitle/PageTitle';

const IndividualHarborPage: React.SFC = ({ children }) => {
  const { t } = useTranslation();
  return (
    <div className={styles.individualHarborPage}>
      <PageTitle title={t('individualHarbor.title')} />
      {children}
    </div>
  );
};

export default IndividualHarborPage;
