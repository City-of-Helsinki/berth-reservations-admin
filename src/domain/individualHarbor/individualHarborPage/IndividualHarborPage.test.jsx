import React from 'react';
import { shallow } from 'enzyme';

import IndividualHarborPage from './IndividualHarborPage';

describe('IndividualHarborPage', () => {
  const getWrapper = (props = { children: 'Individual Harbor Page' }) => shallow(<IndividualHarborPage {...props} />);

  it('renders normally', () => {
    const wrapper = getWrapper();

    expect(wrapper.html()).toMatchSnapshot();
  });
});
