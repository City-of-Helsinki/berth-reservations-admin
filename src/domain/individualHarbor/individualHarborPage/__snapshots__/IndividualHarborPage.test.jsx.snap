// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IndividualHarborPage renders normally 1`] = `"<div class=\\"individualHarborPage\\"><h2 class=\\"pageTitle\\">SATAMA</h2>Individual Harbor Page</div>"`;
